submissions/AndriiZahumnyi/html-css-popup/main_style.css

```diff
@@ -0,0 +1,189 @@
+input, .sub_menu {
+  display: none;
+}
+
+/*position the label*/
+label {
+  position: relative;
+  display: block;
+  cursor: pointer;
+}
+
+/*show the submenu when input is checked*/
+input:checked~.sub_menu {
+  display: block;
+}
+
+                                      /* more Popup menu CSS logic */
+
+#check02, .more_sub_menu {
+  display: none;
+}
+
+/*show the more_sub_menu when input is checked*/
+#check02:checked~.more_sub_menu {
+  display: block;
+}
+
+#check02:checked~.more_behavior {
+  display: none;
+}
+
+                                                      /* Styling of Popup menu */
+
+.head_nav {
+  position: absolute;
+  width: 100%;
+  top: 20px;
+  background-color: ghostwhite;
+}
+
+.container {
+  width: 90%;
+}
+
+.main_menu {
+  display: flex;
+  float: right;
+  flex-direction: row;
+  align-items: center;
+}
+
+ul.main_menu {
+  margin: 5px 0;
+}
+
+ul.main_menu li {
+  list-style-type: none; 
+  padding: 0.5em;
+}
+
+.popup_png {
+  padding: 7.5px;
+}
+
+.popup_png:focus {
+  outline: 2px solid lightblue;
+}
+
+.bell_png {
+  width: 50%;
+}
+
+.kottans_png {
+  width: 125%;
+}
+                        /*position and style main sub_menu, sub_container */
+.sub_menu {
+  position: absolute;
+  width: 300px;
+  height: 500px;
+  background-color: white;
+  top: 90px;
+  right: 15%;
+  border: 1px solid lightgrey;
+  border-radius: 5px;
+  padding: 10px;
+  box-shadow: 5px 10px 18px lightgrey;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.sub_container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  width: 300px;
+  height: 450px;
+}
+
+.sub_menu img {
+  /* width: 70%; */
+  display: block;
+  max-width: 45px;
+  max-height: 45px;
+  padding: auto;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+.sub_menu .align_self:hover {
+  background-color: ghostwhite;
+}
+
+.align_self > img {
+  display: block;
+}
+
+.align_self {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 80px;
+  width: 60px;
+  padding: 0.9em;
+  margin: 2.5px;
+}
+
+.align_self p {
+  margin: 0;
+}
+
+.align_self:focus {
+  outline: 2px solid lightblue;
+}
+
+.sub_menu_footer {
+  width: 100%;
+  background-color: ghostwhite;
+  text-align: center;
+  padding: 7.5px;
+  border-radius: 5px;
+}
+
+                        /*position and style more_sub_menu, more_sub_container */
+
+.more_behavior {
+  text-align: center;
+  border-radius: 5px;
+  background-color: ghostwhite;
+  margin: 1em 0 0 0;
+  padding: 7.5px;
+}
+
+.more_sub_menu hr {
+  background-color: lightblue;
+  height: 1px;
+  border: none;
+  margin: 2.5em -15px;
+}
+
+.more_sub_container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  width: 300px;
+  height: 230px;
+}
+
+.more_sub_menu p {
+  text-align: center;
+}
+
+@media screen and (min-width: 700px) {
+  ul.main_menu li {
+    padding: 1em;
+  }
```

Comment:
I recommend (when your `media` some bigger) add to another file and connect with help `link`